### PR TITLE
Draft Feeder support was implemented

### PR DESCRIPTION
**Pull request by i-nahornyi**

### CODE REVIEW

File: tests/resources/execution_module/action_wrapper.js [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+    }
+    if (iterationCollection.has(hash(value))){
+        var checkCollection = iterationCollection.get(hash(value))
+        if (checkCollection.length <= 0){
+            iterationCollection.set(hash(value),Array.from(value))
+        }
+    }
+    return iterationCollection.get(hash(value)).shift()
+}
+function GetFeederInsideString(value){
```
Comment (zhenyaderyaka) [inline tests/resources/execution_module/action_wrapper.js:5]: The function is pretty the same as GetFeeder. Please try to replace

File: tests/resources/execution_module/action_wrapper.js [hunk 2]
```diff
@@ -43,13 +70,33 @@ module.exports = {
         await driver.findElement(locator).click()
     },
     ExecuteInput: async function (driver, locator, value) {
+        if (typeof value == 'undefined'){
+            throw "Current variable not found, please check your YAML file"
+        }
+        if (typeof value == "object") {
+            if (value == undefined || value == null || value.length <= 0){
+                throw "Variable value not defined, please check your feeder file"
```
Comment (zhenyaderyaka) [inline tests/resources/execution_module/action_wrapper.js:76]: Add variable name to exception

File: tests/resources/execution_module/page_steps_builder.js [hunk 3]
```diff
@@ -1,42 +1,88 @@
-function getInputValue(step){
+function getInputValue(step) {
     return step.value
 }
+function inputValueIsFeeders(value, FEEDERS) {
+    var regexp1 = /^\${(.*?)}$/
```
Comment (zhenyaderyaka) [inline tests/resources/execution_module/page_steps_builder.js]: Doublecheck regular expressions. Seems like they are the same.

**Review by zhenyaderyaka — APPROVED** [review]

File: tests/resources/execution_module/action_wrapper.js (hunk 1 above)
Comment (i-nahornyi) [reply tests/resources/execution_module/action_wrapper.js:5]: function was fixed

File: tests/resources/execution_module/page_steps_builder.js (hunk 3 above)
Comment (i-nahornyi) [reply tests/resources/execution_module/page_steps_builder.js]: code duplication was fixed

**Pull request merged**